### PR TITLE
Cleaning and Updating FROG's UI

### PR DESCRIPTION
**Pull request by AbhinavThukral97**
This PR helps improve the general UI and cleans up some of the existing code: 
- Currently, the colours used for various components are entered explicitly in the styling code making the scheme inconsistent. This PR makes the material colour theme uniform in most of the new components (within imports/ui) as well as several others. Now with a standardised way of setting colour, we can simply update MuiTheme for changes.
- Reduces the shadow/elevation of the components with overpowering shadows (modals, overflow menus, buttons, etc.)
- Styling fixes in Login/Sign up modals. (Links changed to Buttons)
- A couple of changes in the Wizard view. 
- Changes in the Preview toolbar, icons, spacing and some code cleaning
- General fixes in the SideBar spacing, borders, shadows.
- Minor changes in the LandingPage
- A logo placeholder alternate suggestion

### CODE REVIEW

File: frog/imports/client/GraphEditor/SidePanel/index.js [hunk 1]
```diff
@@ -44,7 +44,13 @@ const SideBarHelperText = ({ classes }) => (
...
   <div className="bootstrap">
     <Button
-      style={{ width: '35px', height: '25px' }}
+      style={{
+        width: '40px',
+        height: '30px',
+        border: '1px solid #EEE',
+        margin: '1px',
+        boxShadow: 'none'
+      }}
```
Comment (lfaucon) [inline frog/imports/client/GraphEditor/SidePanel/index.js]: We prefer using makeStyles everywhere than keeping styles as props. If you are refactoring things, don't hesitate to change how styles are done using the preferred way.

Furthermore we would also like to get rid of bootstrap styles and use only MUI everywhere

File: frog/imports/client/Preview/Controls.jsx [hunk 2]
```diff
@@ -176,71 +183,75 @@ export default withStyles(styles)((props: Object) => {
       )}
 
       {storeTemplateFn ? (
-        <div style={{ display: 'flex', alignItems: 'center' }}>
+        <div
+          style={{ display: 'flex', alignItems: 'center', fontSize: '1.25em' }}
```
Comment (lfaucon) [inline frog/imports/client/Preview/Controls.jsx]: Same comment as above. When you refactor styles, change everything to use makeStyles

File: frog/imports/ui/Logo/index.js [hunk 3]
```diff
@@ -5,43 +5,26 @@ import { makeStyles } from '@material-ui/core';
...
+    <circle cx="133" cy="133" r="133" fill="#22A6B3" />
+    <circle cx="180" cy="90" r="10" fill="white" />
+    <path
+      d="M60 172.542C60 167.263 63.6761 162.696 68.8336 161.568L126.365 148.983C133.371 147.45 140 152.786 140 159.958V159.958C140 165.237 136.324 169.804 131.166 170.932L73.6351 183.517C66.6289 185.05 60 179.714 60 172.542V172.542Z"
+      fill="white"
+    />
+    <path
+      d="M65 140.06C65 134.772 68.6865 130.2 73.8542 129.078L191.878 103.457C198.882 101.937 205.5 107.273 205.5 114.44V114.44C205.5 119.728 201.813 124.3 196.646 125.422L78.6223 151.043C71.6184 152.563 65 147.227 65 140.06V140.06Z"
+      fill="white"
+    />
```
Comment (lfaucon) [inline frog/imports/ui/Logo/index.js:27]: Why did you change the logo? :frog:

File: frog/imports/client/GraphEditor/Activities.js [hunk 4]
```diff
@@ -12,7 +12,7 @@ const Box = ({ x, y, width, selected, strokeColor, color }) => (
         x={x - 1}
         y={y - 1}
         width={width + 2}
-        stroke="#31BFAE"
+        stroke="#009688"
```
Comment (lfaucon) [inline frog/imports/client/GraphEditor/Activities.js:15]: You use this color many times. Should it be included in the theme instead? Is there a way to do it?

File: frog/imports/client/GraphEditor/SidePanel/index.js (hunk 1 above)
Comment (AbhinavThukral97) [reply frog/imports/client/GraphEditor/SidePanel/index.js]: Yeah, I believe this was some old code. I'll make the changes to MUI.

File: frog/imports/ui/Logo/index.js (hunk 3 above)
Comment (AbhinavThukral97) [reply frog/imports/ui/Logo/index.js:27]: Just experimenting. I was trying to make a frog symbol from the letter F. It felt more neat. Feel free to revert back though!

Comment (lfaucon) [reply frog/imports/ui/Logo/index.js:27]: It's cool. Keep it this way :)

**Review by lfaucon — APPROVED** [review]
